### ISSUE TITLE
[BUG] [MATRIX] ListItem.DBType / Container.ListItem.DBType are empty/incorrect

### ISSUE THREAD

**Issue by inb4after** [MrKrabat/plugin.video.crunchyroll#44]
Following from: https://github.com/SerpentDrago/skin.auramod/issues/122

The addon does not set the correct DBType at show level/season level/episode level. 

This in turn prevents some controls being visible as they have conditions based on incorrect/missing metadata, and stops context menu items like Kodi's default action "Information" from being displayed.

**Comment by TheFantasticLoki** [comment]
@MrKrabat

**Comment by TheFantasticLoki** [comment]
#45 This fixes the issue for Queue items working on the other categories such as popular & featured.

**Comment by Fatcakes** [comment]
Does this plugin work on Matrix? I haven't updated because I am not sure.

**Comment by MrKrabat** [comment]
> Does this plugin work on Matrix? I haven't updated because I am not sure.

Yes it does work, but you have to use the [Matrix branch](https://github.com/MrKrabat/plugin.video.crunchyroll/tree/matrix).
